### DISCUSSION TITLE
What do people usually do with the default keybindings?

### DISCUSSION THREAD

**Discussion by pufferfish-connoisseur** [wezterm/wezterm#5867] [General]
Is it a common thing to rewrite them from scratch? Or do most people rock with them?

I've gotten to a comfortable spot with neovim, and there they have ample tutorials + kickstart.nvim so that you have nice comfy starting points and grow from there.

I suppose with wezterm that would be the default config, but I've heard mixed stuff about the default keybinds, so I wanted to know what the community thinks.

**Comment by chtenb** [comment]
I disabled the default keybindings and made an alternative scheme that makes sense to me where the modifiers are set to CTRL|SHIFT to minimize conflicts with terminal apps.

**Reply by bew** [reply]
Same, I also disabled the default mouse bindings to reproduce part of the behavior of urxvt I got so accustomed to 🙃

**Comment by MartyMcFlyInTheSky** [comment]
I disabled them, easy enough to reassemble the key ones. Most of my keybindings in the terminal come from bash defaults or bash scripts anyway. I reintroduced only the ones that allow me to create new panes and alter pane size.
